tools/sandbox.py

```diff
@@ -85,3 +86,15 @@ def parse_byte_size(param):
     print(cpu_template)
 uvm.start()
 uvm.get_all_metrics()
+
+kernel_dbg_dir = args.kernel.parent / "debug"
+kernel_dbg = kernel_dbg_dir / args.kernel.name
+print(f"uvm2 with kernel {kernel_dbg} ...")
+uvm2 = vmfcty.build(kernel_dbg, args.rootfs)
+uvm2.spawn()
+uvm2.add_net_iface()
+uvm2.basic_config(vcpu_count=args.vcpus, mem_size_mib=args.guest_mem_size // 2**20)
+uvm2.start()
+# trace-cmd needs this (DNS resolution?)
+uvm2.help.enable_ip_forwarding()
+files = uvm2.help.trace_cmd_guest(["-l", "read_msr"], cmd="sleep 5")
```

Comment:
`sandbox` is a tool to quickly get a microvm and an IPython shell. It is great for quick checks and to bootstrap new tests.

Just type `./tools/devtool sandbox` in a working copy.

Reply:
And the 2nd VM just uses a debug kernel so we can try it out.

Reply:
why not have a `--debug` option to select which vm to boot isntead of booting 2 separate ones?

Reply:
Good idea.

Reply:
Ah thought in that case we won't be able to call trace-cmd always... hmm I'm thinking of leaving it like this for now since this is more for experimentation that anything else.